docs/javascript.md

```diff
@@ -0,0 +1,15 @@
+# JavaScript
+
+RapidRails leverages modern JS libraries and frameworks such as Importmaps, Alpine.js and Hotwire. Here's a brief overview of how each is used:
+
+## Importmaps
+
+Importmaps allow us to control how JavaScript modules are imported in our application. In this project, the configuration for import maps is found in [config/importmap.rb](https://github.com/danielpaul/RapidRails/blob/main/config/importmap.rb). This file contains the mapping of module specifiers to URLs. To find out more see the [importmaps-rails Github](https://github.com/rails/importmap-rails).
```

Comment:
link to official rails guide on how to use import maps and how to add new libraries, etc.